src/Read/DbalFeatureFactory.php

```diff
@@ -12,6 +12,10 @@
 use function array_map;
 use function json_decode;
 
+/**
+ * @psalm-import-type WriteStrategy from \Pheature\Core\Toggle\Write\Strategy
+ * @psalm-import-type DbalFeature from \Pheature\Dbal\Toggle\DbalSchema
```

Comment:
As commented before, I should set here the following:

```suggestion
 * @psalm-type DbalFeature array{feature_id: string, enabled: int, strategies: string}
```